docs/reference/search/profile.asciidoc

```diff
@@ -194,7 +194,7 @@ The API returns the following result:
             "load_source_count": 5
           },
           "debug": {
-            "stored_fields": ["_id", "_ignored", "_routing", "_source"]
```

Comment:
This is not rerturned anymore because the field is not stored anymore.

Reply:
that makes sense, I was expecting this change.